polaris-tokens/src/tokens.ts

```diff
@@ -1,8 +1,7 @@
 import {breakpoints} from './token-groups/breakpoints';
 import {depth} from './token-groups/depth';
 import {legacy as legacyTokens} from './token-groups/legacy';
-import {colors as darkColorScheme} from './token-groups/color.dark';
-import {colors as lightColorScheme} from './token-groups/color.light';
+import {colors} from './token-groups/color';
```

Comment:
```suggestion
import {colors} from './token-groups/colors';
```
**light suggestion (non-blocking)**

Reply:
Should we make them all plural then? Seems like we currently have a mix:

**Singular**: color, depth, motion, shape, zIndex

**Plural**: breakpoints, legacyTokens, spacing (?)

Reply:
I personally have a preference for plural token groups. While our structure is slightly different, I still like to think of token groups as properties on a theme object e.g. `tokens.colors.primary`. It won't work for all token groups, like the ones mentioned above, but I think we should use plural names where possible.